templates/menu--main.html.twig

```diff
@@ -0,0 +1,164 @@
+{#
+/**
+ * @file
+ * Template for the WS Main menu.
```

Comment:
would be good to have here list of main variables in current template
For better future of us